hack/test-cmd.sh

```diff
@@ -191,18 +192,16 @@ atomic-enterprise start \
   --images="${USE_IMAGES}"
 
 # ensure that DisabledFeatures aren't written to config files
-! grep -i '\<disabledFeatures\>' \
-	"${MASTER_CONFIG_DIR}/master-config.yaml" \
-	"${BASETMPDIR}/atomic.local.config/master/master-config.yaml" \
-	"${NODE_CONFIG_DIR}/node-config.yaml"
```

Comment:
@miminar any reason why `NodeConfig` would contain disabled features? The struct [here](https://github.com/openshift/origin/blob/master/pkg/cmd/server/api/types.go#L66) doesn't have any mention of them. I've removed that check.